.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -57,8 +57,7 @@ review this pull request. -->
 This checklist is meant to remind the package maintainer(s) who will review this pull request of some common things to look for. This list is not exhaustive.
 
 - [ ] Do the proposed changes actually accomplish desired goals?
-- [ ] Do the proposed changes follow the [Astropy coding guidelines](https://docs.astropy.org/en/latest/development/codeguide.html)?
-- [ ] Do the proposed changes need the code-style [fixed](https://docs.astropy.org/en/latest/development/workflow/maintainer_workflow.html#pre-commit_bot)?
+- [ ] Do the proposed changes follow the [Astropy coding guidelines](https://docs.astropy.org/en/latest/development/codeguide.html)? Can the code-style be [fixed](https://docs.astropy.org/en/latest/development/workflow/development_workflow.html#pre-commit)?
```

Comment:
I propose we move the code style one to the CI entry, since when the check fails, CI fails, so they are related.